.travis.yml

```diff
@@ -45,11 +45,15 @@ matrix:
     - name: "PHP 7.4"
       php: "7.4"
 
+    - name: "PHP 7.4 (Use Guzzle 6)"
+      php: "7.4"
+      before_script: composer require "guzzlehttp/guzzle:^6.0"
```

Comment:
I haven't tested it extensively, but I believe if we use `--prefer-lowest` then a 5.x release will be pulled in - this is our lowest dependency, and Guzzle's 5.x branch has `"php": ">=5.4"` in its `composer.json` so that we couldn't even trick it by using PHP 7.4 (or 8.0 😅)